src/coreComponents/constitutive/ConstitutiveManager.cpp

```diff
@@ -41,12 +41,14 @@ ConstitutiveManager::~ConstitutiveManager()
 {}
 
 
-Group * ConstitutiveManager::createChild( string const & childKey, string const & childName )
+Group * ConstitutiveManager::createChild( string const & childKey,
+                                          string const & childName,
+                                          bool const allowExistence )
 {
   GEOS_LOG_RANK_0( GEOS_FMT( "{}: adding {} {}", getName(), childKey, childName ) );
   std::unique_ptr< ConstitutiveBase > material =
     ConstitutiveBase::CatalogInterface::factory( childKey, getDataContext(), childName, this );
-  return &registerGroup< ConstitutiveBase >( childName, std::move( material ) );
+  return &registerGroup< ConstitutiveBase >( std::move( material ), allowExistence );
```

Comment:
I also made it so that methods such as these, which register an existing `Group` or `Wrapper` no longer require an additional name (`childName` in this example). In every instance in the code the provided name is the same as the name of the `Group` (or `Wrapper` itself, and it seems this is something we should enforce.